modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/display/context/EditStyleBookEntryDisplayContext.java

```diff
@@ -83,9 +85,10 @@
 public class EditStyleBookEntryDisplayContext {
 
 	public EditStyleBookEntryDisplayContext(
-		HttpServletRequest httpServletRequest, RenderRequest renderRequest,
-		RenderResponse renderResponse) {
+		CETManager cetManager, HttpServletRequest httpServletRequest,
+		RenderRequest renderRequest, RenderResponse renderResponse) {
```

Comment:
Update this display context to pass in _fragmentCollectionContributorRegistry, _frontendTokenDefinitionRegistry, and _itemSelector rather than getting them from the request.